.azure-pipelines.yml

```diff
@@ -0,0 +1,45 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-16.04'
+
+strategy:
+  matrix:
+    Node6:
+      NODE_VERSION: '6.x'
+      TEST_TITLE: 'Node 6 JUnit Test Results'
+    Node8:
+      NODE_VERSION: '8.x'
+      TEST_TITLE: 'Node 8 JUnit Test Results'
+    Node10:
+      NODE_VERSION: '10.x'
+      TEST_TITLE: 'Node 10 JUnit Test Results'
+
+steps:
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: $(NODE_VERSION)
+      checkLatest: true
+    displayName: 'Install Node.js'
+
+  - script: npm install
+    displayName: 'Install dependencies (npm install)'
+
+  - script: npm run test
+    displayName: 'Run tests'
+
+  - script: ./node_modules/.bin/nyc npm test && ./node_modules/.bin/nyc report --reporter=text-lcov | ./node_modules/.bin/coveralls
```

Comment:
I think directly calling something under `./node_modules/` will raise some eyebrows. Let's talk about this